src/Uno.UWP/Storage/StreamedFileDataRequestedHandler.cs

```diff
@@ -0,0 +1,80 @@
+#nullable enable
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Uno;
+using Uno.Extensions;
+
+namespace Windows.Storage
+{
+	public delegate void StreamedFileDataRequestedHandler(StreamedFileDataRequest stream);
+
+	//internal static class StreamedFileDataRequestedHandlerHelper
+	//{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6307767)